po/cs.po

```diff
@@ -26,147 +26,143 @@ msgstr "Odeslat heslo zpět do systému"
 
 #: ../src/core/org.freedesktop.systemd1.policy.in.in.h:2
 msgid "Authentication is required to send the entered passphrase back to the system."
-msgstr "Autentizace je vyžadována pro odeslání zadaného hesla do systému."
+msgstr "Pro odeslání zadaného hesla do systému je vyžadováno ověření."
 
 #: ../src/core/org.freedesktop.systemd1.policy.in.in.h:3
 msgid "Manage system services or other units"
-msgstr "Správa systémových služeb nebo dalších jednotek"
+msgstr "Spravovat systémové služby nebo další jednotky"
```

Comment:
Google translate offers another variant "Správa systémových služeb nebo jiných disků" which is actually closer to the original translation. This is of course not at all proof that it sounds good ☺ (sometimes it is the contrary), but I'm curious what's wrong with the old one and what's the nature of changing these existing translations?